### ISSUE TITLE
Q: can we add a "demo" dataset?

### ISSUE THREAD

**Issue by breznak** [dbolya/yolact#281]
As the COCO dataset is quite large, can we create and include a dummy demo dataset (say cats-dogs)? 
This would demonstrate the folder structure, JSON files and labes, bounding boxes. Ideally this would be only 10s train samples, and the network would train on it (really simple). 

Would you be for including that?

**Comment by dbolya** [comment]
I agree that would be useful, so I'll add it to the TODO. However, I'm not sure YOLACT would do well on such a tiny dataset so making a training configuration for it would be kinda rough.

**Comment by breznak** [comment]
> I agree that would be useful, [...] not sure YOLACT would do well on such a tiny dataset

yes, that's a problem. I have a simulator that creates the images as well as all the needed annotations, so I can generate a dataset we need. Ofc it would really be more for "how the COCO dataset should be structured", and results would likely overfit. 

Would something like 1000 train, 300 eval, 300 test images be reasonable to include in the repo?

**Comment by breznak** [comment]
We might add the dataset to torch-vision instead and use it from there? (Already dependency) 
https://pytorch.org/docs/stable/torchvision/datasets.html

**Comment by dbolya** [comment]
Interesting proposition with the torchvision datasets. The dataloader currently wouldn't be able to support that, but it looks like they have both COCO and Pascal SBD, so it might be better just to switch to their format. It would make dataset management a whole lot easier if they could manage it.

Your proposed skew sounds fine, and maybe we can use resnet-18 or something to reduce the possibility of overfitting. Also, someone who made a collab demo for evaluation said they were also going to make one for training. This dummy dataset would be great for super fast training on collab given the quick timeouts there.
